lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -30,66 +30,146 @@ class MinitestAssertions < Base
           RESTRICT_ON_SEND = %i[
             assert_equal
             assert_not_equal
-            refute_equal
+            assert_includes
+            assert_not_includes
```

Comment:
Does it exist?

Reply:
It's defined by Rails: https://github.com/rails/rails/blob/main/activesupport/lib/active_support/test_case.rb#L206